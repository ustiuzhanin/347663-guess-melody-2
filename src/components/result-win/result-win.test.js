import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import ResultWin from "./result-win.jsx";

test(`ResultWin's snapshot`, () => {
  const tree = renderer.create(
      <BrowserRouter>
        <ResultWin time={10} errors={1} onClick={jest.fn()} />
      </BrowserRouter>
  );

  expect(tree).toMatchSnapshot();
});
