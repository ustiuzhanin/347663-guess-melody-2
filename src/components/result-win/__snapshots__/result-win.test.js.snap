// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultWin's snapshot 1`] = `
<section
  className="login"
>
  <div
    className="login__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="login__title"
  >
    You are an actual music lover!
  </h2>
  <p
    className="login__total"
  >
    For 
    0
     minutes and 
    10
     seconds you\`ve scored
     
    2
     points, and made 
    1
     mistakes
  </p>
  <a
    className="replay"
    href="/"
    onClick={[Function]}
  >
    Try Again!
  </a>
</section>
`;
